app.js

```diff
@@ -0,0 +1,17 @@
+const express = require ('express')
+const app = express()
+const {getApi, getTopics} = require('./apiController')
+
+
+
+app.get("/api", getApi)
+
+app.get('/api/topics', getTopics)
+
+app.use((req, res, next) => {
```

Comment:
If this is for the invalid endpoints, we can use app.all() rather than app.use() here - this will catch all requests (GET, POST PATCH etc) that don't get caught by the above endpoints, and then we pass "*" as the first argument - so that it matches ANY endpoint that isn't one of the above. The second argument to the app.all would be this same callback function that you've created here, sending the 404 - the app.use() functions are better to make use of for passing on things like errors - see for the syntax on this: https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro